src/components/media_control/media_control.js

```diff
@@ -644,6 +644,16 @@ export default class MediaControl extends UIObject {
     this.unbindKeyEvents()
   }
 
+  /**
+   * enables to configure the media control after its creation
+   * @method configure
+   * @param {Object} options all the options to change in form of a javascript object
+   */
+  configure(options) {
+    this._options = $.extend(this._options, options)
```

Comment:
Can we clone only Media Control options, to avoid duplication of unnecessary options inside Media Control? 
Something like:  `this._options = $.extend(this._options, options.mediacontrol)`

Reply:
I agree, however I think I would need help since I am not that familiar with this codebase and it appears that not all `MediaControl`-specific options are namespaced under `options.mediacontrol` (for example, `options.hideMediaControl`, and presumably also `options.hideVolumeBar`).

Likewise, it seems like [`Container.configure`](https://github.com/clappr/clappr/blob/147867b33c9bcd1268aff6505bbbdb33d2a01e62/src/components/container/container.js#L463) should also only be called if container-specific options are changed. Perhaps [`Core.configure`](https://github.com/clappr/clappr/blob/147867b33c9bcd1268aff6505bbbdb33d2a01e62/src/components/core/core.js#L338) should look like this:
```javascript
...
this.containers.forEach((container) => {
    if (containerSpecificOptionsChanged) {
        container.configure(this.options)
    }
})
if (mediaControlSpecificOptionsChanged) {
    this.mediaControl.configure(this.options)
}
...
```